core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -864,10 +864,14 @@ class SQLPlanParserSuite extends BaseTestSuite {
         val (eventLog, _) = ToolTestUtils.generateEventLog(eventLogDir,
           "ProjectExprsSupported") { spark =>
           import spark.implicits._
-          val df1 = Seq(9.9, 10.2, 11.6, 12.5).toDF("value")
+          val df1 = Seq((9.9, "ABC"), (10.2, "abc"), (11.6, ""), (12.5, "AaBbCc"))
+                      .toDF("num", "str")
+          // write df1 to parquet to transform LocalTableScan to ProjectExec
```

Comment:
I prefer to create a separate unit-test for the `translate`.

- Modifying an existing unit-test to test for something different is recommended because we will never know if the code change actually breaks the original unit test or not.
- There are other expressions that are part of projectExec; each of which has its own unit-test. Otherwise, we will have a single unit-test for all expressions that could appear under projectExec

Reply:
Thanks! Test is updated.